doc/manual/50-feature-details.en.md

```diff
@@ -539,6 +539,24 @@ Retention policy based on recovery window
     backups required to allow point-in-time recovery back to 9:30 AM
     on the previous Friday.
 
+> **IMPORTANT:**
+> Incremental backups are not considered during the retention policy
+> process. This is because incremental backups depend on either a full
+> backup or other incremental backups to be valid. To maintain the
+> consistency of backup chains, only full backups are reviewed when
+> applying the retention policy.
```

Comment:
```suggestion
> Block-level incremental backups are not accounted during the retention
> policy processing. This is because this kind of incremental backups
> depends on all of its parent backups up to the full backup which generates
> the chain in order to be recoverable. To maintain the consistency of backup
> chains, only full backups are taken into account when applying retention
> policies.
```

Reply:
LGTM! @gcalacoci ?

Reply:
LGTM

Reply:
I'm updating the suggestion.